plugin/declarativenetworkmanager.cpp

```diff
@@ -1,3 +1,11 @@
+/*
+ * Copyright © 2025 Jolla Mobile Ltd
```

Comment:
Precisely being also 2024 here and probably most of the other places too. The company instance is the same even with the name changes.

Reply:
Not going to turn back the clock. This change was started in 2025.

Reply:
Yea but the file and the work related was created in 2024.

Reply:
The name change is done in 2025 as well. Jolla Mobile Ltd did not exist in 2024. We cannot backdate the copyrights like this. It should have the old name then for older changes.